Source/WebCore/crypto/keys/CryptoKeyOKP.cpp

```diff
@@ -98,7 +98,7 @@ RefPtr<CryptoKeyOKP> CryptoKeyOKP::importJwk(CryptoAlgorithmIdentifier identifie
         if (keyData.crv != "Ed25519"_s)
             return nullptr;
         // FIXME: Do we have tests for these checks ?
-        if (!keyData.alg.isEmpty() && keyData.alg != "EdDSA"_s)
+        if (!keyData.alg.isEmpty() && (keyData.alg != "EdDSA"_s && keyData.alg != "Ed25519"_s))
```

Comment:
Does WPT have coverage for different casing of this argument? E.g., `ed25519` presumably fails?

Reply:
Good point ! I think there isn't coverage for that case in the current tests. I'll add them as part of this patch.

Reply:
Done